examples/basic-video.html

```diff
@@ -95,6 +101,13 @@
       // Called by Networked-Aframe when connected to server
       function onConnect () {
         console.log("onConnect", new Date());
+
+        // Handle camera button click (Off and On)
+        cameraBtnEle.addEventListener('click', function() {
+          NAF.connection.adapter.enableCamera(!cameraEnabled);
+          cameraEnabled = !cameraEnabled;
+          cameraBtnEle.innerHTML = cameraEnabled ? 'Camera Mute' : 'Camera Unmute';
```

Comment:
Same comment applies here for `.innerHTML`
Instead of mute/unmute, `Hide Camera/Show Camera` may be better?

Reply:
yes correct